buildSrc/build.gradle

```diff
@@ -123,6 +123,8 @@ dependencies {
   api 'org.jruby.joni:joni:2.2.1'
   api "com.fasterxml.jackson.core:jackson-databind:${props.getProperty('jackson_databind')}"
   api "org.ajoberstar.grgit:grgit-core:5.2.1"
+  api "org.bouncycastle:bc-fips:1.0.2.5"
```

Comment:
Version 1.0.2.4 appears to be the latest approved for FIPS

Reply:
I was not aware of this, I'll downgrade ASAP

Reply:
actually 2.0.0 is out now :fire: